three things:
1. we don't want builder being counted
2. it's illegal in some countries to send information anywhere without asking the user
3. pld builders shouldn't do any network traffic

--- initng-0.5.2/Makefile.am~	2006-01-14 11:29:46.000000000 +0200
+++ initng-0.5.2/Makefile.am	2006-01-19 00:28:40.000000000 +0200
@@ -27,7 +27,7 @@
 	@echo "# /sbin/ngc -c"
 	@echo
 	@echo
-	./count_me.sh
+	echo ./count_me.sh
 
 initng-paths.h:		$(top_builddir)/config.status
 	@echo '#ifndef INITNG_INITNG_PATHS_H'  >$@
